delphin/cli/edm.py

```diff
@@ -47,6 +50,17 @@ def call_compute(args):
     print(f'  F-score:\t{f}')
 
 
+def eds_from_mrs(m: mrs.MRS, predicate_modifiers=True, skip_on_errors=True) -> eds.EDS:
+    try:
+        e = from_mrs(m, predicate_modifiers=True)
```

Comment:
Given the `predicate_modifiers` parameter above, I think you meant `predicate_modifiers=predicate_modifiers` here? But again, I think the function parameter can just be removed unless we actually want to configure this.